lib/modules/keyboardNav.js

```diff
@@ -1194,7 +1195,17 @@ function reply(selected = getSelected()) {
 	));
 }
 
+let npEscapeActive = false;
```

Comment:
Not entirely sure why this variable is necessary; can't you just use `goModeActive` directly? Also, I think it's worth considering not checking for `goMode` at all, if it would always make sense to escape NP mode.

Reply:
Initially the reason was that goModeActive was being set to false before navigateTo() was being called in handleCallbacks(), so I made a more delayed boolean (it switches to false only after all callbacks have been called). I could have changed the order that things were handled in handleCallbacks(), but I wanted to avoid causing any unintentional side-effects. 

The only thing with always escaping NP mode is that, if the user changes their mind and exits go mode (or was never in go mode), I need some check to ensure that NP mode isn't being escaped, because navigateTo() is also being used in other non-go-mode functions.